test/parallel/test-net-connect-timeout.js

```diff
@@ -15,9 +15,11 @@ var gotConnect = false;
 
 var T = 100;
 
-
-// 240.*.*.*.* is "reserved for future use"
-var socket = net.createConnection(9999, '240.0.0.0');
+// 192.0.2.1 is part of subnet assigned as "TEST-NET" in RFC 5737.
+// For use solely in documentation and example source code.
+// In short, it should be unreachable.
+// In practice, it's a network black hole.
+var socket = net.createConnection(9999, '192.0.2.1');
```

Comment:
If this were a socket we were opening for listening or connecting to on the local server, I'd say so. But this is a port on a non-existent remote server. Using it here would seem to confer no advantage and would have the downside of introducing murkiness as to the purpose of `common.PORT` (which is to make sure that tests run in parallel do not try to listen or connect on the same port on the local machine at the same time).